go/pkg/client/cas.go

```diff
@@ -469,33 +486,33 @@ func (c *Client) UploadIfMissing(ctx context.Context, data ...*uploadinfo.Entry)
 // * Do we want to allow []byte uploads, or require the user to construct Chunkers?
 // * How to consistently distinguish in the API between should we use GetMissing or not?
 // * Should BatchWrite be a public method at all?
-func (c *Client) WriteBlobs(ctx context.Context, blobs map[digest.Digest][]byte) error {
+func (c *Client) WriteBlobs(ctx context.Context, blobs map[digest.Digest][]byte) ([]digest.Digest, int64, error) {
```

Comment:
We don't really need to change the interface of these middle-layer functions, right? We only use UploadIfMissing in the top layer. Ditto below.